### PR TITLE
Add anonymous-credentials review to wasabi/research-club

### PR DESCRIPTION
**Pull request by markon1-a**
This PR adds [anonymous-credentials](https://www.youtube.com/watch?v=pgErjQSQQsg) transcript review to the wasabi/research-club directory.

### CODE REVIEW

File: wasabi/research-club/anonymous-credentials.md [hunk 1]
```diff
@@ -1,695 +1,327 @@
...
-Thank you.
-Did you finish?
-
-Speaker 0: 00:05:31
-
-Yeah, that's almost everything I wanted to say.
-Like the other thing I wanted to say is that these tokens are pretty flexible compared to just Schnorr signatures.
-So at the Building on Bitcoin Conference 2018 I talked about how to use this to basically swap a token that is in or that it was created with an anonymous credential with an on-chain or off-chain Bitcoin on Lightning without the server knowing that this happened and without having to trust the other party.
+Jonas  5:31  
+Yeah, that's almost everything I wanted to say. Like, the other thing I wanted to say is that these tokens are pretty flexible compared to just Schnorr signatures. So at the building on Bitcoin Conference, Unknown speaker018, I talked about how to use this to basically swap a token that is in or that is most created with an anonymous credential with an on chain or off chain Bitcoin on lightning without the server knowing that this happened and without having to trust the other party.
```
Comment (kouloumos) [inline wasabi/research-club/anonymous-credentials.md:33]: There is a random "Unknown speaker" text inside of this text

File: wasabi/research-club/anonymous-credentials.md [hunk 2]
```diff
@@ -1,695 +1,327 @@
...
-Yeah, so mathematically, I think the best way to think about this is as Peterson multi-commitments.
-So in normal Peterson commitments, right, you have some randomness and you use that to commit to a value.
-And you basically have two different generators, group generators.
-Now with these multi-commitments used in some of these credential approaches, you have randomness and now you commit to multiple attributes, not only a single one.
-And Usually you have as many generators as you have attributes.
-So I think historically one of the first notions of these credentials was by Branz.
-I think Peter Branz, not quite sure.
-He wrote his PhD.
-Yeah.
+Jonas  3:Unknown speaker5
```
Comment (kouloumos) [inline wasabi/research-club/anonymous-credentials.md:26]: You deleted the previous timestamps and altered the original formatting.

File: wasabi/research-club/anonymous-credentials.md [hunk 3]
```diff
@@ -1,695 +1,327 @@
...
-And the trick is that you could actually delegate the ownership of the credential to someone else.
-So in our case, that would be, I think, giving money to someone and I wouldn't be able to redeem it, but you would, which is actually pretty nice.
-Oh my God, I wasn't talking.
-
-Speaker 0: 00:09:26
+Speaker 1  8:03  
+Yes, it's called I think, reissuance or that's what usually comes up. Anyway. So a bit more there. Because the same outdoors were working on this for probably a decade of this paper and, and they actually came up with something called mercury, our signatures where they don't need so much crypto as much as in this paper. But what in a very straightforward way they could achieve the exact same thing with this mercury as signatures. And even more. They were working on some things called delegated anonimous credentials, and they're the three keys that you could actually delegate the ownership of the credential to someone else. So in our case, that would be I think, giving money to someone, and I wouldn't be able to redeem it, but you would, which is actually pretty nice. Oh, my God, I wasn't talking?
```
Comment (kouloumos) [inline wasabi/research-club/anonymous-credentials.md:46]: You correctly catch the wrongly transcribed "Ray Schwentz" and fixed it to  "reissuance" but for the rest of this text you introduced a bunch of mistakes. For example:
- "authors" to "outdoors"
- "Mercurial Signatures" to "mercury, our"
- "and the trick is that" to "and the're the three keys that"

**Review by kouloumos — COMMENTED** [review]
Welcome @markon1-a  and thank you for your first contribution! I’ve taken some time to review your submission and would like to share some observations and seek clarification to ensure we’re aligned with the project’s standards.

Firstly, I noticed that the original formatting has been altered, which significantly complicates the evaluation process. Our review guidelines emphasize maintaining the original formatting for consistency and readability. Could this deviation be due to the guidelines not being clear, or was there a difficulty in accessing them? Understanding your perspective will help us improve our guidelines or their visibility.

Additionally, it appears there have been instances where, instead of correcting transcription errors, new mistakes were introduced. I'm curious if this resulted from an automated review process or another method that might have contributed to these additions. To assist you, I've highlighted specific examples in my inline comments, aiming to provide clarity on the type of errors to watch out for.

I also observed the insertion of "Unknown speaker" throughout the transcript without consistent formatting, and at times, it seems to have been added arbitrarily. Adhering to formatting guidelines is crucial for maintaining the transcript’s structure and clarity.

We truly value the effort and time every reviewer invests in their submissions. Our goal is not only to enhance the quality of the content but also to support our reviewers in understanding and meeting the project's expectations. It’s essential that the review process is approached with diligence to ensure the integrity and usefulness of the transcripts.

To support you further and answer any questions you might have, I’d like to extend an invitation [to join our Discord community](https://discord.gg/76wCxgeyvM). It’s a great space to connect with other contributors, seek advice, and share insights.

I’m here to help and would like to understand the challenges you faced with this submission better. Your feedback is crucial for us to provide the right support and guidance. We’re looking forward to your reply and to assisting you in any way we can.

File: wasabi/research-club/anonymous-credentials.md (hunk 1 above)
Comment (markon1-a) [reply wasabi/research-club/anonymous-credentials.md:33]: My bad I put that there while I was editing.

File: wasabi/research-club/anonymous-credentials.md (hunk 2 above)
Comment (markon1-a) [reply wasabi/research-club/anonymous-credentials.md:26]: Was I suppose to edit in the same job area?

File: wasabi/research-club/anonymous-credentials.md (hunk 3 above)
Comment (markon1-a) [reply wasabi/research-club/anonymous-credentials.md:46]: How do I make changes?

Comment (kouloumos) [reply wasabi/research-club/anonymous-credentials.md:46]: You can do the changes through the website editor in the same way that you initially reviewed and submitted the transcript.

File: wasabi/research-club/anonymous-credentials.md (hunk 2 above)
Comment (kouloumos) [reply wasabi/research-club/anonymous-credentials.md:26]: I am not sure that I understand your question. Based on the review guidelines you should keep the original formatting of the timestamps.

**Comment by kouloumos** [comment]
Closing this based on what we discussed privately.

**Pull request closed without merging**